src/features/authPlaygroundAPI/authPlaygroundApiSlice.ts

```diff
@@ -0,0 +1,16 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+import { ArkReauthQuery } from "../../lib/authentication/arkBaseQuery";
+
+const baseurl = "";
+
+export const authPlaygroundApi = createApi({
+  reducerPath: "authPlayground",
+  baseQuery: ArkReauthQuery(baseurl),
```

Comment:
ArkReauthQuery needs also the instance to get the Token.
@AndreaCuneo any suggestion? can be viable to pass it from the invoking component?